package.json

```diff
@@ -40,10 +40,11 @@
     "ember-click-outside-modifier": "^2.0.0",
     "ember-concurrency": "^2.1.0",
     "ember-element-helper": "^0.5.5",
-    "ember-focus-trap": "^0.7.0",
     "ember-set-helper": "^2.0.1",
     "ember-truth-helpers": "^3.0.0",
-    "tracked-maps-and-sets": "^3.0.1"
+    "focus-trap": "^6.7.1",
+    "tracked-maps-and-sets": "^3.0.1",
+    "ember-auto-import": "^1.11.2"
```

Comment:
I moved this dep (`ember-auto-import`) here, because `focus-trap` is not imported correctly when `ember-auto-import` was in dev.